contracts/MiniMeToken.sol

```diff
@@ -488,7 +493,8 @@ contract MiniMeToken is Controlled {
             _cloneTokenName,
             _cloneDecimalUnits,
             _cloneTokenSymbol,
-            _transfersEnabled
+            _transfersEnabled,
+            l1Address
```

Comment:
We omit the docstring comment for `_l1Address`.

Reply:
The `l1address` isn't passed to the enclosing function.